### PR TITLE
Ship Class/Spec

### PR DESCRIPTION
**Pull request by willfox0409**
**Pull Request #1**

Created Ship class

- initialized ship object with attributes
- implemented #sunk? and #hit methods
- created comprehensive tests, all tests passing
- perhaps there are still edge cases to review?
- any necessary changes can be approached as we go or through an official refactor

### CODE REVIEW

File: lib/ship.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Ship
+    attr_reader :name, :length, :health
+
+    def initialize(name, length)
+        @name = name
+        @length = length 
+        @health = length 
+    end
+
+    def sunk?
```
Comment (Coralruschak) [inline lib/ship.rb:10]: great problem solving with this method
